### DISCUSSION TITLE
Examples for extra_styles and base_config?

### DISCUSSION THREAD

**Discussion by emufan** [jcwillox/lovelace-paper-buttons-row#76] [Q&A] [answered]
Can I find examples for the new and some old options anywhere? Would be very helpful, because they sound very promising, but currently I don't have ideas for best practices, etc.

e.g.

styles with esp. new extra_styles
built-in animations for blink and rotating
base_config
hide_state and hide_badge

**Comment by jcwillox** [comment, marked as answer]
Yeah, I wasn't motivated enough at the time to go through and add examples for all the new stuff, but someone specifically asking for them is good enough motivation so I'll look at adding those tomorrow.

If you want some immediate examples checkout button-cards documentation for `extra_styles` and animations as they are almost identical.

I thought there were some examples that use `base_config` but simply you define a base button config that is merged with every button, it's mainly useful so you can define a base style that applies to all buttons, then override parts of it on a per button basis.

Edit: README has been updated

**Reply by emufan** [reply]
No need to have uit tomorrow, but nevertheless very welcome, it it'll come at one point in time. 👍
